.github/release-workflows.jsonnet

```diff
@@ -94,4 +106,43 @@ local imagePrefix = 'grafana';
       },
     },
   }),
+  'images.yml': std.manifestYamlDoc({
+    name: 'publish images',
+    on: {
+      push: {
+        branches: [
+          'k[0-9]+*',  // This is a weird glob pattern, not a regexp, do not use ".*", see https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet
+          'main',
+          'remove-drone',
```

Comment:
Is this for testing, or do we need a follow-on PR to remove "remove-drone"

Reply:
just for testing: https://github.com/grafana/loki/actions/runs/11040346516/job/30668118522